### PR TITLE
docs(aio): Add Alain Chautard in GDE list

### PR DESCRIPTION
**Pull request by alcfeoh**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -591,5 +591,14 @@
     "website": "http://ngnir.life/",
     "bio": "Nir is a Principal Frontend Consultant & Head of the Angular department at 500Tech, Google Developer Expert and community leader. He organizes the largest Angular meetup group in Israel (Angular-IL), talks and teaches about front-end technologies around the world. He is also the author of two books about Angular and the founder of the 'Frontend Band'.",
     "group": "GDE"
+  },
+  
+  "achautard": {
+    "name": "Alain Chautard",
+    "picture": "https://avatars1.githubusercontent.com/u/4699771?s=400&u=3289780ac60d0dbb65f9c112d898236ec02799f1&v=4",
```
Comment (Splaktar) [inline aio/content/marketing/contributors.json]: 👋 

You should add your picture to https://github.com/angular/angular/tree/master/aio/content/images/bios

**Review by Splaktar — CHANGES_REQUESTED** [review]
You probably want to put `docs(aio): ` in front of your commit message and change `Added` to `add`.

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (alcfeoh) [reply aio/content/marketing/contributors.json]: My bad it actually works on my branch :-)

**Comment by alcfeoh** [comment]
Everything should be good now. Uploaded pic and link to the pic.

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (Splaktar) [reply aio/content/marketing/contributors.json]: Yep, a branch from your fork is all that you need 😄

**Comment by Splaktar** [comment]
Content looks good, but you probably want to squash the commits and update the commit message as I mentioned [here](https://github.com/angular/angular/pull/23783#pullrequestreview-118614287).

**Review by StephenFluin — APPROVED** [review]

**Review by StephenFluin — DISMISSED** [review]
Content looks good, but we need to fix the commit message & squash the commits.

**Comment by alcfeoh** [comment]
I changed the commit name, tried to squash which seems to be successful, and just in case this can also be done while merging in one button click (squash and merge)

**Comment by StephenFluin** [comment]
I'm seeing 5 commits now, so it looks like your squash didn't work. We can't use squash and merge from GitHub because it doesn't do things in a clean way.

**Comment by alcfeoh** [comment]
Ok I'll do a new PR then, it's probably the easiest way to proceed

**Comment by alcfeoh** [comment]
When I don't use Github, I create a branch and try to push my changes, here is what I'm getting @StephenFluin :
```
achautard-patch:refs/heads/achautard-patch 
Pushing to https://github.com/angular/angular
remote: Permission to angular/angular.git denied to alcfeoh.
```

which brings me back to my initial question about being added to the contributor list.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
@alcfeoh, I fixed it up for you.
(For future reference: You had to update the branch on your fork (https://github.com/alcfeoh/angular), not https://github.com/angular/angular. By updating the branch that it associated with this PR (`patch-1`), the PR is updated automatically.)

**Comment by alcfeoh** [comment]
@gkalpak Thanks a lot.

**Pull request closed without merging**